routing/router.go

```diff
@@ -1141,7 +1143,8 @@ func (r *ChannelRouter) processUpdate(msg interface{}) error {
 
 		// If the channel doesn't exist in our database, we cannot
 		// apply the updated policy.
-		if !exists {
+		isStaleUpdate := time.Since(msg.LastUpdate) > r.cfg.ChannelPruneExpiry
```

Comment:
This looks like new behavior (other than zombie check)?

I think it would be worth documenting what is going on here, returning different error messages (see comment above), and possibly make this its own commit (or describe in the commit message what this is good for).

Reply:
Fixed.